quodlibet/data/operon.bash

```diff
@@ -21,6 +41,9 @@ _operon() {
     local cur prev words cword
     _init_completion -n = || return
 
+    local audio_extensions='@(a?(l)ac|ac3|ape|flac|m[4k]a|mid|mp3|og[ag]|opus|w?(a)v)'
+    local image_extensions='@(jp?(e)g|png)'
```

Comment:
Same question here. This list is used to show image files for the `image-set` command of operon

Reply:
gif comes to mind

Reply:
ok, added